madre10 project

martindreymann opened the pull request.

martindreymann committed:
Created folder

Xrosby reviewed (approved):
What a good job. I see you put some effort into this! Looking forward to see more of your hard work

martindreymann committed:
Create .gitignore

martindreymann committed:
Simple implementation of requirements assignment 1

martindreymann committed:
MariaDB + php.ini file + login/logout/register

martindreymann committed:
Preparing for image upload

martindreymann committed:
Can upload images..

martindreymann committed:
register and upload

martindreymann committed:
Upload, feed, comment, login, logout, register complete

XSS is still very much possible. No security whatsoever. No layout either really.

martindreymann committed:
santization

martindreymann committed:
Ajax

martindreymann committed:
super secure password recovery

martindreymann committed:
changed DB name to avoid collisions

martindreymann committed:
Screenshots of page

Xrosby reviewed (approved):
Seems to fulfill all the requirements.

martindreymann committed:
Update login.php
